redis_rate_limit/__init__.py

```diff
@@ -43,21 +43,23 @@ class RateLimit(object):
     This class offers an abstraction of a Rate Limit algorithm implemented on
     top of Redis >= 2.6.0.
     """
-    def __init__(self, resource, client, max_requests, expire=None, redis_pool=REDIS_POOL):
+    def __init__(self, resource, client, max_requests, whitelisted_clients=[], expire=None, redis_pool=REDIS_POOL):
```

Comment:
We should think about a better name here since there's a movement to avoid some terms in software development. Maybe `ignore_clients` or `bypass_clients`.

Reply:
Rename to ignored_clients=None

Reply:
Done